pkgs/applications/misc/onagre/default.nix

```diff
@@ -27,14 +35,35 @@ rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "sha256-IOhAGrAiT2mnScNP7k7XK9CETUr6BjGdQVdEUvTYQT4=";
```

Comment:
should we switch this to `cargoHash`

Reply:
Maybe, what would the difference here be?

Reply:
i think there has been some change where `hash` is preferred instead of using `sha` if possible.
i just figured, we can add it to the commit while you're at it.

although, I cannot find the RFC or the specific documentation that outlines this change but lately committers point out in their reviews to use `hash` instead of `sha` in my PRs

Reply:
It seems the hash is not used at all, since we vendor the whole lockfile, can you confirm that on your end? Maybe it is just a caching step.

I removed `sha/hash`.